### PR TITLE
Display consistent params for generic calls

### PR DESCRIPTION
**Pull request by MiltonTulli**
# Description

Show the parameters in a simple way once initially expanding generic call actions.

Closes https://github.com/DXgovernance/DAVI/issues/207

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

# Checklist:

- [ ] My code follows the existing style of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any UI changes have been tested and made responsive for mobile views

## Video

https://user-images.githubusercontent.com/25844967/189235418-7bd61481-8750-4526-a545-70eea998073a.mov

### CODE REVIEW

**Comment by Filipv95** [comment]
@MiltonTulli Looks good to me.

**Comment by rossneilson** [comment]
One failing test, looks like snapshots need updated

**Comment by MiltonTulli** [comment]
> One failing test, looks like snapshots need updated

Fixed

**Comment by rossneilson** [comment]
Hey, I think some slight misunderstanding here. The idea with the designs was to have a new section for this nice formatted section and then the user can still press the function name button to see the raw params. Let me know if you have more questions but my bad this wasn't clear. 
Screenshot 2022-09-06 at 09 51 04

**Comment by MiltonTulli** [comment]
> Hey, I think some slight misunderstanding here. The idea with the designs was to have a new section for this nice formatted section and then the user can still press the function name button to see the raw params. Let me know if you have more questions but my bad this wasn't clear. Screenshot 2022-09-06 at 09 51 04

@rossneilson This is ok for you? 
I think is a lot of information (approval + vest) but that was the way I found to make it readable. We could try putting approval section at the bottom or putting vest "intearct with" part at the top. Let me know what do you think. 
Screen Shot 2022-09-06 at 19 59 26

**Comment by rossneilson** [comment]
> @rossneilson This is it ok for you? I think it is a lot of information (approval + vest) but that was the way I found to make it readable. We could try putting approval section at the bottom or putting vest "intearct with" part at the top. Let me know what you think. Screen Shot 2022-09-06 at 19 59 26

Hmmm, it will be a lot of information on the spending call ones. I think putting `Interact with X` -> `All readable params` -> `function button` -> Repeat for next call.
Not sure if this makes sense, if not we can jump on a call.

**Comment by Filipv95** [comment]
@MiltonTulli Looks good to me, great job!

**Comment by rossneilson** [comment]
It would be nice to have this behaviour similar across all of our action types, just testing it seems like transfer doesn't have the same behaviour as generic calls

**Comment by MiltonTulli** [comment]
> It would be nice to have this behaviour similar across all of our action types, just testing it seems like transfer doesn't have the same behaviour as generic calls

Sounds good! Since this task had several changes after I started working on it I would prefer to create new issues for all action types also to avoid making this larger.

**Review by dcrescimbeni — CHANGES_REQUESTED** [review]

File: src/locales/en/translation.json [hunk 1]
```diff
@@ -225,13 +225,12 @@
   "pageNotExist": "That page doesn't exist",
   "makeSureCorrectAddress": "Make sure you typed the correct address",
   "stakeTokens": "Stake {{token}} tokens",
-  "ipfs": {
-    "failedToUpload": "Failed to upload to IPFS, please refresh the page and try again",
-    "hashNotTheSame": "Hash returned by pinata isn't the same as local hash",
-    "errorWhileUploading": "Error while uploading to IPFS",
-    "couldntUploadToPinata": "Couldn't upload and pin to Pinata"
-  },
```
[the remark this answers is not in the document]
Comment (MiltonTulli) [reply src/locales/en/translation.json:233]: Very weird, i don't remember me removing this. Thanks for bringing this up. 
I restored ipfs translations. Btw I think we are not using `failedToUpload` and `couldntUploadToPinata` keys.

**Review by dcrescimbeni — APPROVED** [review]

**Pull request merged**